src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Bookstore from './components/Bookstore';
+// import { Provider } from 'react-redux';
```

Comment:
[ OPTIONAL BUT RECOMMENDED] Please remove commented code as it will make the whole code unclean and inefficient.